spread_tests/main/ap/task.yaml

```diff
@@ -0,0 +1,58 @@
+summary: Test default configuration at the beginning,
+    stop daemon, start ap by hand and see wlan1 can connect to it
```

Comment:
the daemon is not actually stopped. Perhaps you can say "put wifi-connect in manual mode"